### PR TITLE
app-admin/consul-bin: init at 1.7.2

### PR DESCRIPTION
**Pull request by maxeaubrey**

### CODE REVIEW

File: app-admin/consul-bin/consul-bin-1.7.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+DESCRIPTION="Cluster manager from Hashicorp"
+HOMEPAGE="https://www.consul.io"
+SRC_URI="https://releases.hashicorp.com/${MY_PN}/${PV}/${MY_PN}_${PV}_linux_amd64.zip -> ${P}.zip"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=""
```
Comment (65278) [inline app-admin/consul-bin/consul-bin-1.7.2.ebuild]: I'd wager this depends on glibc.

File: app-admin/consul-bin/consul-bin-1.7.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils user
```
Comment (65278) [inline app-admin/consul-bin/consul-bin-1.7.2.ebuild]: Can't quite fathom what we need eutils for here.

File: app-admin/consul-bin/consul-bin-1.7.2.ebuild (hunk 1 above)
Comment (65278) [reply app-admin/consul-bin/consul-bin-1.7.2.ebuild]: Disregard that, it's statically linked.Wow.

File: app-admin/consul-bin/consul-bin-1.7.2.ebuild (hunk 2 above)
Comment (maxeaubrey) [reply app-admin/consul-bin/consul-bin-1.7.2.ebuild]: Correct, but i don't think this really hurts? Can rebase and fix it here and for nomad-bin too, if you'd like.

**Pull request merged**